workers/unity/Assets/Fps/Scripts/GameLogic/Respawning/SpawnPoints/SpawnPointIndicator.cs

```diff
@@ -4,11 +4,12 @@ namespace Fps
 {
     public class SpawnPointIndicator : MonoBehaviour
     {
-        private readonly Vector3 boxDimensions = new Vector3(1, 0, 1);
+        private static readonly Color transparentLightGreen = new Color(.1f, 1f, .1f, .8f);
```

Comment:
I'd not use the color name as variable name here, but something more generic. `gizmoColor`